### ISSUE TITLE
Latest update seems to make MS Office apps crash

### ISSUE THREAD

**Issue by RincewindTheWizard** [tekezo/Karabiner#779]
Since last Karabiner upgrade (a few days ago) MS Office 2016 applications crash every time I use certain combinations including the Command key (example: Command-Enter to send an email on Outlook, or Command-+). The Command keys are remapped as follows:
   KANA to Command_R
   EISUU to Command_L
and either of the two causes the same effect.
Before the latest upgrade, everything was working just fine.

Update 1: with the newest Office update (v15.29, downloaded this morning) Command-Enter doesn't make Outlook crash any longer, just it doesn't do anything (that is, it doesn't send the email). MS Word keeps crashing, including with Command-Enter.

Update 2: I just downloaded and installed Karabiner 10.21.0 (which used to work fine with all Office apps) and the problem is exactly the same as with 10.22.0.

It looks to me like some weird interaction between Karabiner and Office - it started with a Karabiner upgrade, didn't change with an Office upgrade, and stays even with Karabiner previous version. But with the previous versions of both Karabiner and Office it would work just fine. 

Anyone having similar issues?

MacBook Pro (Retina, 13-inch, Early 2015) with Japanese keyboard
OSX v. 10.11.6 (El Capitan)
Karabiner 10.22.0

Thank you for your support,
Alfonso

**Comment by tekezo** [comment]
Could you confirm whether this issue will be solved turning on this checkbox in Karabiner Preferences?
![161214-0001](https://cloud.githubusercontent.com/assets/659178/21177382/47063356-c22e-11e6-808c-97d3fee81ba0.png)

**Comment by RincewindTheWizard** [comment]
Same as before on v10.21.0 – re-upgrading to v10.22.0, will let you know in minutes.

**Comment by RincewindTheWizard** [comment]
No change even with 10.22.0, sorry.

**Comment by tekezo** [comment]
I guess it is MS Office issue if the AXNotifier is not the cause.
Is this issue not happen without Karabiner?

**Comment by RincewindTheWizard** [comment]
Yes, if I close Karabiner all is normal – it definitely does not happen without Karabiner...

**Comment by RincewindTheWizard** [comment]
By the way, there are combinations that make Word crash but not Outlook, and others that make Chrome crash (also this never happened before). For example:

 

Command-Enter: Word crashes, Outlook no action (it should send the current draft instead), Chrome crashes

Command-Backspace(delete): Word crashes, Outlook no action, Chrome crashes

Command-C:  Word OK, Outlook OK, Chrome crashes (but not with Command-V)

 

Looks quite weird, but this is what is going on on my laptop right now... yesterday I reset the PRAM just in case, but nothing changed.

 

From: Alfonso BONGIORNO <alfonso.bongiorno@gmail.com>
Date: Thursday, 15 December 2016 at 08:39
To: tekezo/Karabiner <reply@reply.github.com>, tekezo/Karabiner <Karabiner@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: Re: [tekezo/Karabiner] Latest update seems to make MS Office apps crash (#779)

 

Yes, if I close Karabiner all is normal – it definitely does not happen without Karabiner...
